include/seqan3/alphabet/quality/all.hpp

```diff
@@ -51,22 +53,25 @@
  *
  * ###Encoding Schemes
  *
- * | Format                      | Quality Type          | Phred Score Range  | Rank Range   | ASCII Range  | Assert                    |
- * |:---------------------------:|:----------------------|:------------------:|:------------:|:------------:|:-------------------------:|
- * | Sanger, Illumina 1.8+ short | seqan3::phred42       | [0 .. 41]          | [0 .. 41]    | ['!' .. 'J'] | Phred score in [0 .. 61]  |
- * | Sanger, Illumina 1.8+ long  | seqan3::phred63       | [0 .. 62]          | [0 .. 62]    | ['!' .. '_'] | Phred score in [0 .. 62]  |
- * | Solexa, Illumina [1.0; 1.8[ | seqan3::phred68legacy | [-5 .. 62]         | [0 .. 67]    | [';' .. '~'] | Phred score in [-5 .. 62] |
+ * | Format                      | Quality Type          | Phred Score Range | Rank Range | ASCII Range  | Assert                    |
+ * |:---------------------------:|:----------------------|:-----------------:|:----------:|:------------:|:-------------------------:|
+ * | Sanger, Illumina 1.8+ short | seqan3::phred42       | [0 .. 41]         | [0 .. 41]  | ['!' .. 'J'] | Phred score in [0 .. 61]  |
+ * | Sanger, Illumina 1.8+ long  | seqan3::phred63       | [0 .. 62]         | [0 .. 62]  | ['!' .. '_'] | Phred score in [0 .. 62]  |
+ * | Solexa, Illumina [1.0; 1.8[ | seqan3::phred68legacy | [-5 .. 62]        | [0 .. 67]  | [';' .. '~'] | Phred score in [-5 .. 62] |
+ * | PacBio                      | seqan3::phred94       | [0 .. 62]         | [0 .. 93]  | ['!' .. '~'] | Phred score in [0 .. 93]  |
```

Comment:
This is not a Pacbio format. It is actually also `Sanger, Illumina 1.8+ ` but the full range. The other two are both short in that sense

Reply:
should I rename `long` into `middle` when I rename `PacBio` into `Sanger, Illumina 1.8+ full range`?